src/components/DataTable/cells/TableCellChip.tsx

```diff
@@ -0,0 +1,21 @@
+import {TableCellWrapper} from './TableCellWrapper';
+import {Badge} from '~/index';
+
+type Props = {
+    value: string;
+};
+
+export const TableCellChip = ({value}: Props) => {
+    if (!value) {
+        return <TableCellWrapper>-</TableCellWrapper>;
+    }
```

Comment:
As we use this condition in all `TableCell`, maybe it can be a good idea to manage it in `TableCellWrapper`